packages/util/src/helpers.ts

```diff
@@ -1,5 +1,21 @@
 import { isHexString } from 'ethjs-util'
 
+/**
+ * Checks if a number string is a hex string, if not, converts and returns it 
+ * @param {string} input string to check, convert, and return
+ */
+ export const toHex = function (input?: string) {
```

Comment:
trying to think of a less ambiguous name, what do you think of `numberToHex`?

```suggestion
 export const numberToHex = function (input?: string) {
```